ddtrace/contrib/grpc/patch.py

```diff
@@ -15,8 +15,6 @@
     distributed_tracing_enabled=True,
 ))
 
-# TODO[tbutt]: keeping name for client config unchanged to maintain backwards
-# compatibility but should change in future
```

Comment:
any reason to remove this? seems unrelated to this PR

Reply:
I thought @majorgreys was alluding to these future `DD_SERVICE` changes.. in which case the future _is now_ :smile: If not I can put it back if we have other plans for the client service name.

Reply:
oh, it should still change in the future :)